metrics/core/src/main/java/org/hyperledger/besu/metrics/opentelemetry/OpenTelemetrySystem.java

```diff
@@ -151,6 +151,7 @@ private Object extractValue(final MetricData.Type type, final MetricData.Point p
       final String name,
       final String help,
       final String... labelNames) {
+    LOG.trace("Creating a counter");
```

Comment:
Should we add counter name as a logging arg?  Same for timer and gauge.

Reply:
Done